build/images/Dockerfile.build.controller.ubi

```diff
@@ -33,7 +33,7 @@ RUN --mount=type=cache,target=/go/pkg/mod/ \
     --mount=type=cache,target=/root/.cache/go-build/ \
     make antrea-controller
 
-FROM ubuntu:22.04
+FROM registry.access.redhat.com/ubi9
```

Comment:
@tnqn @antoninbas  can we get this PR merged? I think at least the base image here should be changed to UBI. This is probably introduced during the image splitting.

Reply:
This is a good catch, I will also patch the 1.15 release ASAP